var/spack/repos/builtin/packages/parallel-netcdf/package.py

```diff
@@ -12,9 +12,14 @@ class ParallelNetcdf(AutotoolsPackage):
     Unidata's NetCDF."""
 
     homepage = "https://trac.mcs.anl.gov/projects/parallel-netcdf"
```

Comment:
I would change the homepage to https://parallel-netcdf.github.io/. The previous homepage says:

> The official project web pages and source code repository of PnetCDF have been migrated to github... The old project main page is kept below and will be removed in the near future.